indexers/vector/FaissIndexer/__init__.py

```diff
@@ -25,6 +26,7 @@ def __init__(self,
                  index_key: str,
                  train_filepath: Optional[str] = None,
                  distance: str = 'l2',
+                 normalize: bool = True,
```

Comment:
I think is better to keep it False since it is a transformation of the input

Reply:
Yes, you're right.